c/example04.c

```diff
@@ -0,0 +1,154 @@
+/* Example 04 :
+This example shows how to create and delete a tag for a specified resource.
+*/
+
+#include <string.h>
+#include <stdlib.h>
+#include <curl/curl.h>
+
+#define AK_SIZE 20
+#define SK_SIZE 40
+
+struct Tag {
+    char *_resources_id;
```

Comment:
The `_` before the variable name, is in theory reserved for compiler in C (and C++) , and should be avoided. (see here https://www.iso-9899.info/n1570.html#7.1.3)
also, doesn't rally make sense, just keep key, value... :)